### PR TITLE
Add Riot.im

### PR DESCRIPTION
**Pull request by jtagcat**

### CODE REVIEW

**Comment by Carlgo11** [comment]
GitHub should really have a scratchpad.
I'll just leave this here temporarily so that I know what parts of the PR that I've already reviewed.

```
Note to maintainers and myself:

Alexa ranking and category is ok
Name: "Riot"
Doc link might not be allowed if tfa = no. I'll have to check that tomorrow.
Facebook: RiotChat
Double blank lines at end.
```

File: _data/communication.yml [hunk 1]
```diff
@@ -180,6 +180,14 @@ websites:
       software: Yes
       doc: https://peerio.zendesk.com/hc/en-us/articles/202933829-How-do-I-setup-two-factor-authentication-
 
+    - name: Riot.im
+      url: https://about.riot.im/
+      img: riot.png
+      twitter: RiotChat
+      tfa: No
+      doc: https://github.com/vector-im/riot-web/issues/2772
```
Comment (Carlgo11) [inline _data/communication.yml]: So I ran the PR locally and `doc` links are ignored when tfa is set to No.


Therefore, please remove the doc link:
```suggestion
```

File: _data/communication.yml [hunk 2]
```diff
@@ -180,6 +180,14 @@ websites:
...
+
```
Comment (Carlgo11) [inline _data/communication.yml]: Double blank lines.
```suggestion
```

File: _data/communication.yml [hunk 3]
```diff
@@ -180,6 +180,14 @@ websites:
       software: Yes
       doc: https://peerio.zendesk.com/hc/en-us/articles/202933829-How-do-I-setup-two-factor-authentication-
 
+    - name: Riot.im
```
Comment (Carlgo11) [inline _data/communication.yml]: On their website they use "Riot" as their name.
```suggestion
    - name: Riot
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I suggest you also add their facebook handle "RiotChat".
`facebook: RiotChat`

**Comment by kmpoppe** [comment]
@vc7ee I might have overlooked, but did you add the `facebook`-tag as @Carlgo11 suggested?

**Review by kmpoppe — DISMISSED** [review]
Was sceptical about `status` with a non-Twitter-URL (because never seen before) but it compiles and works just fine. Thanks again for your contribution @vc7ee!

**Comment by Carlgo11** [comment]
The status tag is supposed to be set when a company is actively working on implementing 2FA. That way the Twitter message changes from "Please support two factor authentication" to "Thank you for working on two factor authentication".

From what I read on https://github.com/vector-im/riot-web/issues/2772 it doesn't sound like they're actively working on it.

**Comment by jtagcat** [comment]
How do you 'request review from *'?

**Comment by Carlgo11** [comment]
That’s a Maintainer only feature.

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**